torch/csrc/distributed/rpc/rpc_command_base.h

```diff
@@ -11,7 +12,11 @@ class RpcCommandBase {
  public:
   // Need to override this to serialize the RPC. This should destructively
   // create a message for the RPC (Hence the &&).
-  virtual Message toMessage() && = 0;
+  Message toMessage() && {
+    JitRRefPickleGuard jitPickleGuard;
+    return std::move(*this).toMessageInternal();
+  }
+  virtual Message toMessageInternal() && = 0;
```

Comment:
This should be called `toMessageImpl`, `Internal` kind of indicates some internal details of the class and shouldn't be a public interface method.